conf.in.py

```diff
@@ -220,18 +221,38 @@
 locale_dirs = ["../i18n/"]
 gettext_compact = False
 
+try:
+    user_paths = [Path(p) for p in os.environ["PYTHONPATH"].split(os.pathsep)]
+except KeyError:
+    user_paths = []
+
 class_maps = {}
-for module in ("3d", "analysis", "core", "gui", "server"):
-    with open(f"/usr/lib/python3/dist-packages/qgis/{module}/class_map.yaml") as f:
-        class_maps[module] = yaml.safe_load(f)
+qgis_module_path = None
+for path in user_paths:
+    candidate_path = path / "qgis" / "core"
+    if candidate_path.exists():
+        qgis_module_path = candidate_path
+        break
+
+if qgis_module_path:
```

Comment:
this will be silently skipped, should we at least raise a warning?
I'm a bit concerned that for whatever reason, there is no more class_map file from QGIS and the docs are built without error.

Reply:
Ok, I've reverted that bit, but have revised the class map download to use a user-writable location instead.